game first attempt

anton-kuvaldin opened the pull request:
Запуск:
1) ./main.py

Управление:
1) перемещеие:  ← → ↑ ↓

Выход:
По нажатию крестика в окне

Commit:
game first attempt

febos reviewed (commented):
При запуске получил такое:
pygame 1.9.4
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "/home/baulin/Desktop/PythonFIVT/Review1/Kuvaldin/python_classes-dev/main.py", line 33, in <module>
    walk = [pygame.image.load(os.path.join(image_path, 'spacecraft0.png')), pygame.image.load(os.path.join(image_path, 'spacecraft1.png')),
pygame.error: Couldn't open /home/antony/python/MyGame/test/resources/images/spacecraft0.png

1) Избавьтесь от абсолютной адресации
2) import * использовать не рекомендуется, исправьте
3) приведите код к PEP8 (как минимум есть строки длиннее 79 символов)

anton-kuvaldin committed:
Add files via upload

anton-kuvaldin committed:
Add files via upload

anton-kuvaldin committed:
Create a

anton-kuvaldin committed:
Add files via upload

anton-kuvaldin committed:
Delete a

anton-kuvaldin committed:
Update howto.md

anton-kuvaldin committed:
Update howto.md

anton-kuvaldin committed:
Update howto.md

anton-kuvaldin committed:
Update main.py

anton-kuvaldin committed:
all

1) Исправил нежелательный import *
2) Подогнал под pep8 за исключением некоторых случаев, где это, как мне показалось, глупо

anton-kuvaldin committed:
PEP8

1) Подогнал под PEP8 за исключением некоторых мест, где это, как мне показалось, глупо
2) Избавился от нежелательных import *
